src/components/Chart/Maps.js

```diff
@@ -189,6 +227,56 @@ export class GenericMap extends Component {
       })
     })
   }
+  renderPointTooltip() {
+    const {
+      hoverPoint,
+      layout: {
+        projectPoint,
+        numberFormat
+      }
+    } = this.state
+    const {
+      width,
+      pointLabel,
+      pointDetails,
+      unit,
+    } = this.props
+    if (!hoverPoint) {
+      return null
+    }
+    const [ x, y ] = projectPoint([hoverPoint.datum.lon, hoverPoint.datum.lat])
+
+    const value = isNaN(hoverPoint.datum.value) 
+      ? String(hoverPoint.datum.value).trim()
+      : numberFormat(hoverPoint.datum.value)
+
+    const body = pointDetails.map(t => {
+      const val = hoverPoint.datum[t]
+      if (val) {
+        return (<>{`${t}: ${ isNaN(val) ? val : numberFormat(+val) }`}<br/></>)
```

Comment:
```suggestion
        return (<>{subsup(val)}<br/></>)
```

We should not force the `Column: Value` format and also no number formatting. Just assume that the details are strings and if they contain numbers they are correctly pre formatted.